core/src/test/groovy/com/novoda/gradle/release/TestGeneratePomTask.groovy

```diff
@@ -0,0 +1,140 @@
+package com.novoda.gradle.release
+
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
+
+class TestGeneratePomTask {
```

Comment:
This test stucks for ever...

Reply:
Sometimes it also happened to me. Restart the test fixed it. No idea why...